### PR TITLE
fancier 404 page

### PR DESCRIPTION
**Pull request by local-girl**
Possibly need a german graphic ?

### CODE REVIEW

**Comment by nidico** [comment]
I think the image should be explicit `` instead of background and the `{{code}} - {{message|translate}}` part in the `alt` tag.

**Comment by nidico** [comment]
I could live without a German version of the graphics now, as this is only shown exceptionally. We can improve that later on.

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/TopLevelState/templates/Error.html [hunk 1]
```diff
@@ -1 +1,4 @@
-<h1>{{code}} - {{message | translate}}</h1>
+<div class="error-page error-page-{{code}}" data-ng-switch="code">
+    <img data-ng-switch-when="404" src="/static/images/404.png" style="margin: 0 auto" alt="{{code}} - {{message | translate}}" />
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/TopLevelState/templates/Error.html]: style should not be in the template

File: src/mercator/static/stylesheets/scss/a3.scss [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 @import "compass/css3";
 @import "compass/css3/inline-block";
+@import "compass/typography/text/replacement";
```
Comment (xi) [inline src/mercator/static/stylesheets/scss/a3.scss]: No longer needed.

**Comment by xi** [comment]
I also added some default HTTP error messages.

**Pull request merged**